index.html

```diff
@@ -215,11 +215,11 @@ <h2>About Myself</h2>
                     Get my resume
                 </button>
```

Comment:
> - [ ] Based on the request made by the previous code reviewer, it appears that you haven't included the button pressed states in your **"Get my resume"** and **"See project"** buttons. It would be greatly appreciated if you could incorporate the necessary interaction for these states. Doing so will enhance the user experience and contribute to a more engaging website. Keep up the good work!👍